src/server/qgshttprequesthandler.cpp

```diff
@@ -505,6 +511,8 @@ bool QgsHttpRequestHandler::startGetFeatureResponse( QByteArray* ba, const QStri
     format = "text/xml";
 
   setInfoFormat( format );
+  startStreamingResponse();
```

Comment:
@AsgerPetersen yeah, sorry I just had a quick look and it seemed to me that you added `startStreamingResponse();` inside `sendResponse()` that means for **ALL** request including WMS etc.